### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by R-Ellis-84**
Updated README.md to include information about the software and how to use it.

Added quick setup section, and provided links to references for contributing and other Quebec_v2 documentation.  Added list of dependencies and reference to requirements.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,99 @@
...
+### Data Setup
+Place properly formated JSON files in a file directory you can access from a terminal or cli.  
+> [!NOTE]
+> You will need the file path to the stock.json file to upload your data to the container.
 
 
 ### Upload specific portfolio data
 
 To upload your portfolio data use the command below:
     `docker run -v /HOST/PATH: /CONTAINER/PATH -it <container_name>`
```
Comment (mintfish3r) [inline README.md]: it should be <image_name> after `-it`

File: README.md [hunk 2]
```diff
@@ -1,73 +1,99 @@
...
+example:
+    `docker run -v /Users/lindseybaucum/Desktop/stock.json:/quebec_v2/stock.json -it m1ntfish3r/quebec:amd`
 
 
 ### Running the Application
-To price an equity portfolio from root directory:
 
-`python3 driver.py`
+Once the container is running the price.py program can be run from the root directory.
+`python3 PriceApp/price.py
```
Comment (mintfish3r) [inline README.md]: I believe it should be `python3 -m PriceApp.price`

File: README.md [hunk 3]
```diff
@@ -1,73 +1,99 @@
...
+
+> [!NOTE]
+> To run the quebec_v2 test suite and ensure the application is performing as expected run `pyest`.
+To price an equity portfolio from root directory:
```
Comment (mintfish3r) [inline README.md]: Content is missing after the colon

File: README.md [hunk 4]
```diff
@@ -1,15 +1,137 @@
...
+* Docker
+* Additional dependencies [requirements.txt](https://github.com/kgp33/quebec_v2/blob/main/requirements.txt)
 
-### Diagrams
+## Diagrams
 To view the diagrams in VSCode, you'll need to install the PlantUML extension in VSCode to render them. After install:
 
 1. Open file
 2. Right click anywhere in the code
 3. Click "Preview Current Diagram
```
Comment (mintfish3r) [inline README.md]: missing right quotation mark

File: README.md (hunk 1 above)
Comment (R-Ellis-84) [reply README.md]: Changed to: `docker run -v /HOST/PATH: /CONTAINER/PATH -it image_name`

File: README.md (hunk 2 above)
Comment (R-Ellis-84) [reply README.md]: changed to python3 -m PriceApp.price

File: README.md (hunk 3 above)
Comment (R-Ellis-84) [reply README.md]: Removed "To price..."

File: README.md (hunk 4 above)
Comment (R-Ellis-84) [reply README.md]: Added the right "

**Review by mintfish3r — APPROVED** [review]
Looks good.

**Pull request merged**